master/internal/taskconfigpolicy/task_config_policy.go

```diff
@@ -0,0 +1,41 @@
+package taskconfigpolicy
```

Comment:
nit: what do you think about using `package policies` and using `config_policies` for the directory name? Not a big deal either way; just a little more concise and I think we usually separate words with an underscore for directories. There's probably already a config package... and I think go prefers short package names?

Reply:
I like `configpolicies`! I think that's a great balance. What do you think?

Reply:
Agreed! Changed